openslides_backend/models/base.py

```diff
@@ -21,11 +22,18 @@ def __new__(metaclass, class_name, class_parents, class_attributes):  # type: ig
             metaclass, class_name, class_parents, class_attributes
         )
         if class_name != "Model":
+            new_class.field_prefix_map = {}
```

Comment:
Where do we use this new field_prefix_map? I can not find it in the code.

Also: Why do we want to save the template fields twice, once with the pythonic attribute name (e. g. `group__ids` for `User` and once with a shortend name (e. g. `group_`)?

Reply:
I would not call it magic, but yes, thank you for the docs :D my bad for that.